app/prj.conf

```diff
@@ -207,3 +208,21 @@ CONFIG_NPM1300_CHARGER=y
 # Task Watchdog
 CONFIG_TASK_WDT=y
 CONFIG_TASK_WDT_CHANNELS=6
+
+# Memfault
+CONFIG_MEMFAULT=y
+CONFIG_MEMFAULT_NCS_PROJECT_KEY="key"
+CONFIG_MEMFAULT_HTTP_PERIODIC_UPLOAD=y
+CONFIG_MEMFAULT_ROOT_CERT_STORAGE_NRF9160_MODEM=y
```

Comment:
This eats quite a bit off flash as certs are kept in flash. We might need to change this at some point to require certs to be already provisioned